pss/outbox/outbox.go

```diff
@@ -151,3 +184,7 @@ func (o *Outbox) requeue(slot int) {
 func (o *Outbox) len() int {
 	return cap(o.slots) - len(o.slots)
 }
+
```

Comment:
CurrentSize() and len() have the same function bodies and CurrentSize is used only in tests. CurrenSize can be removed.